integrations/optimum/pydoc/config.yml

```
@@ -0,0 +1,32 @@
+loaders:
+  - type: haystack_pydoc_tools.loaders.CustomPythonLoader
+    search_path: [../src]
+    modules:
+      [
+        "haystack_integrations.components.embedders.optimum_backend",
+        "haystack_integrations.components.embedders.optimum_document_embedder",
+        "haystack_integrations.components.embedders.optimum_text_embedder",
+        "haystack_integrations.components.embedders.pooling",
+      ]
+    ignore_when_discovered: ["__init__"]
+processors:
+  - type: filter
+    expression:
+    documented_only: true
+    do_not_filter_modules: false
+    skip_empty_modules: true
+  - type: smart
+  - type: crossref
+renderer:
+  type: haystack_pydoc_tools.renderers.ReadmePreviewRenderer
+  excerpt: Optimum integration for Haystack
+  category_slug: haystack-integrations
```

Apologies, we just recently changed the required category slug.

```suggestion
  category_slug: integrations-api
```